src/extension.ts

```diff
@@ -31,6 +31,65 @@ export function activate(context: vscode.ExtensionContext) {
 	});
 	context.subscriptions.push(disposable_number_cursors_from_one);
 	
+	let disposable_number_cursors_from_arbitrary = vscode
+			.commands
+			.registerTextEditorCommand('engineernick.multi-cursor-tools.number_cursors_from_arbitrary', (editor, edit, args) => {
+		vscode.window.showInputBox({
+			placeHolder: "1",
+			prompt: "Starting number",
+			value: 1
+		}).then((num)=>{
+			if(parseFloat(num).toString() == num){
+				editor.edit((edit)=>{
+					for (let selection of editor.selections){
+						edit.replace(selection,num.toString());
+						num++;
+					}
+				});
+			}else{
+				console.error(num);
+			}
```

Comment:
It is probably just intended behaviour for nothing to happen, better not to log an error? I think many other extentions might also log errors and it would just get lost, only helpful during development.
```suggestion
			}
```

Reply:
Yeah makes sense ... it was indeed for debugging purposes ... I left them for the sole purpose that I'm not familiar with what kind of information you get as an extension author when people report bugs ... but thought it could include the console logs/errors generated by said extension ... so could help with debugging/helping users ... 

So I removed them ! :-)